### PR TITLE
Fix conflict #11689 - Fix missing hooks on warehousecard

### PR DESCRIPTION
**Pull request by wdammak**
@eldy 
@atm-greg 
https://github.com/Dolibarr/dolibarr/pull/11689

### CODE REVIEW

File: htdocs/product/stock/card.php [hunk 1]
```diff
@@ -86,79 +89,30 @@
...
 {
-	$object->fetch(GETPOST('id', 'int'));
-	$result=$object->delete($user);
-	if ($result > 0)
+				 
+													
+ 
+	// Ajout entrepot
+	if ($action == 'add' && $user->rights->stock->creer)
+
```
Comment (eldy) [inline htdocs/product/stock/card.php]: Why adding so many spaces ?

Comment (wdammak) [reply htdocs/product/stock/card.php]: Sorry! Notepad...
Fixed

**Comment by wdammak** [comment]
@eldy : Continuous Integration Test failed ?
the next step?

**Comment by wdammak** [comment]
@eldy & @atm-greg c'est quoi le problème d'après vous pour proposer des corrections

**Comment by atm-maxime** [comment]
Pull develop, error seems to come from a test file

**Pull request closed without merging**